pnpm-lock.yaml

```diff
@@ -4449,6 +4479,18 @@ packages:
       slash: 3.0.0
     dev: true
 
+  /@jest/console/29.1.2:
```

Comment:
It'd be nice if you could match the versions of the packages being used by other projects in the monorepo, so we don't wind up with multiple versions all over the place.

Reply:
Done in e4f05e7ef1.

Reply:
... and 6b5d85334f because I had to merge in a renovate-bot update from trunk. The merge is unverified because... it's a merge. Merges aren't the right place to fix linting issues.

Reply:
... and 12f6cea9d3 because the lockfile needed updating post merge....